index.js

```diff
@@ -1,26 +1,36 @@
-var path        = require('path')
-var loaderUtils = require('loader-utils')
+var path = require('path');
+var glob = require('glob');
+var loaderUtils = require('loader-utils');
 
-module.exports = function(source) {
+module.exports = function (source) {
+    var query = loaderUtils.parseQuery(this.query);
+    var manifest = query.manifest || (query.manifestPath ? requireManifests(query.manifestPath) : false);
 
-  var query = loaderUtils.parseQuery(this.query)
-  var manifest = query.manifest || (query.manifestPath ? requireManifest(query.manifestPath) : false)
-
-  if (manifest) {
-    for ( var key in manifest ) {
-      var value = manifest[key]
-      var pattern = new RegExp(key, 'gm')
-      source = source.replace(pattern, value)
+    if (manifest) {
+        for (var key in manifest) {
+            var value = manifest[key];
+            var pattern = new RegExp(key, 'gm');
+            source = source.replace(pattern, value);
+        }
     }
-  }
 
-  return source
+    return source;
+};
+
+function requireManifests(manifestGlob) {
+    return glob.sync(manifestGlob).map(function (path) {
+        return requireManifest(path);
+    }).filter(function (manifest) {
+        return manifest !== false;
+    }).reduce(function (manifest, current) {
+        return Object.assign(manifest, current);
+    }, {});
 }
 
 function requireManifest(manifestPath) {
-  try {
-    return require(manifestPath)
-  } catch(error) {
-    return false
-  }
+    try {
+        return require(process.cwd() + path.sep + manifestPath);
+    } catch (error) {
+        return false;
+    }
 }
```

Comment:
Did this go from 2 to 4 spaces? Do you mind reformatting it back to 2 spaces?

Reply:
I'm used to 4 spaces indentation, but you're right - I shouldn't have changed such a thing in one's project.

As of version, this PR is not a breaking change, so I don't think major version should be bumped, but it's up to you.

Reply:
I might be the one to have this project transferred to, no problem here :)